basics/basics.md

```diff
@@ -2,9 +2,17 @@
 
 ## Why do we need OlympusDAO in the first place?
 
-Dollar-pegged stablecoins have become an essential part of crypto due to their lack of volatility as compared to tokens such as Bitcoin and Ether. Users are comfortable with transacting using stablecoins knowing that they hold the same amount of purchasing power today vs. tomorrow. But this is a fallacy. The dollar is controlled by the US government and the Federal Reserve. This means a depreciation of dollar also means a depreciation of these stablecoins.
-
-OlympusDAO aims to solve this by creating a non-pegged stablecoin called OHM. By focusing on supply growth rather than price appreciation, OlympusDAO hopes that OHM can function as a currency that is able to hold its purchasing power regardless of market volatility.
+Dollar-pegged stablecoins have become an essential part of crypto due to their
+lack of volatility as compared to tokens such as Bitcoin and Ether. Users are
+comfortable with transacting using stablecoins knowing that they hold the same
+amount of purchasing power today vs. tomorrow. But this is a fallacy. The dollar
+is controlled by the US government and the Federal Reserve. This means a
+depreciation of dollar also means a depreciation of these stablecoins.
+
+OlympusDAO aims to solve this by creating a backed free floating reserve
+currency called OHM. By focusing on supply growth rather than price
```

Comment:
```suggestion
OlympusDAO aims to solve this by creating a free-floating reserve currency, OHM,
that is backed by a basket of assets. By focusing on supply growth rather than price
```

Reply:
Emphasize that it is backed by a basket of assets